cypher/cypher-docs/src/docs/dev/ql/administration/indexes-for-search-performance/index.asciidoc

```diff
@@ -184,6 +184,28 @@ ON (r.propertyName)
 
 Index provider can be specified using the `OPTIONS` clause.
```

Comment:
Add something about b-tree deprecated here as well, similar to how was done for drop by schema